migrations/20210506023749-add-budget-references.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    /**
+     * Add altering commands here.
+     *
+     * Example:
+     * await queryInterface.createTable('users', { id: Sequelize.INTEGER });
+     */
+    await queryInterface.addConstraint('Budgets', {
```

Comment:
This addConstraint call adds the foreign key reference from the UserId column in Budgets to the id column in Users.